src/queues/sqs/notification/test.rs

```diff
@@ -243,3 +244,127 @@ fn serialize_complaint_feedback_type() {
+                        "emailAddress":"bar@example.com"
+                    }
+                ],
+                "reportingMTA": "example.com",
+                "timestamp":"2012-05-25T14:59:38.605Z",
+                "feedbackId":"000001378603176d-5a4b5ad9-6f30-4198-a8c3-b1eb0c270a1d-000000",
+                "remoteMtaIp":"127.0.2.0"
+            }
+        },
+        {
+            "notificationType": "Complaint",
+            "mail": {
+                "timestamp":"2018-02-14T09:15:51 +0000",
+                "messageId":"002",
+                "source":"qux@example.com",
+                "sourceArn": "arn:aws:ses:us-west-2:888888888888:identity/example.com",
+                "sourceIp": "127.0.2.0",
+                "sendingAccountId":"2345",
+                "destination":[
+                    "baz@example.com"
+                ],
+                "headersTruncated":false,
+                "headers":[ 
+                    { 
+                        "name":"From",
+                        "value":"\"Qux\" <qux@example.com>"
+                    }
+                ],
+                "commonHeaders":{
+                    "subject":"Complaint Subject"
+                }
+            },
+            "complaint": {
+                "userAgent":"AnyCompany Feedback Loop (V0.01)",
+                "complainedRecipients":[
+                    {
+                        "emailAddress":"baz@example.com"
+                    }
+                ],
+                "complaintFeedbackType":"abuse",
+                "arrivalDate":"2009-12-03T04:24:21.000-05:00",
+                "timestamp":"2012-05-25T14:59:38.623Z",
+                "feedbackId":"000001378603177f-18c07c78-fa81-4a58-9dd1-fedc3cb8f49a-000000"
+            }
+        },
+        {
+            "notificationType": "Delivery",
+            "mail": {
+                "timestamp":"2017-04-24T15:44:00 +0000",
+                "messageId":"003",
+                "source":"quuz@example.com",
+                "sourceArn": "arn:aws:ses:us-west-2:888888888888:identity/example.com",
+                "sourceIp": "127.0.3.0",
+                "sendingAccountId":"3456",
+                "destination":[
+                    "xyzzy@example.com"
+                ],
+                "headersTruncated":false,
+                "headers":[ 
+                    { 
+                        "name":"From",
+                        "value":"\"Quuz\" <quuz@example.com>"
+                    }],
+                "commonHeaders":{
+                    "subject":"Delivery Subject"
+                }
+            },
+            "delivery": {
+                "timestamp":"2014-05-28T22:41:01.184Z",
+                "processingTimeMillis":546,
+                "recipients":["xyzzy@example.com"],
+                "smtpResponse":"250 ok:  Message 64111812 accepted",
+                "reportingMTA":"a8-70.smtp-out.amazonses.com",
+                "remoteMtaIp":"127.0.2.0"
+            }
+        }]
```

Comment:
I read about a cool trick the other day that may or may not be handy here: [include_str!](https://doc.rust-lang.org/std/macro.include_str.html)

Basically we could put this big blob of JSON into a file in the same directory and call it something like `test_data.json` (or whatever you think makes sense), then `include_str!` reads it into a `&str` for us in one step.

Just keeps this file a bit smaller and makes things like editing easier, because you're editing a `.json` file so editors and IDEs know how to behave etc.